sass/sass_wrapper.js

```diff
@@ -0,0 +1,46 @@
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
+if (typeof global.gc !== 'function') {
+  require('v8').setFlagsFromString('--expose_gc');
+  global.gc = require('vm').runInNewContext('gc');
+}
+
+function main(args) {
+  if (runAsWorker(args)) {
+    log('Starting Sass compiler persistent worker...');
```

Comment:
Should this be `debug` instead of `log`?

Reply:
Done.